packages/answers-kit/src/utils/url.ts

```diff
@@ -1,5 +1,44 @@
 import type { SortingOptions } from '@/types.js';
 
+/**
+ * Append query parameters to URL
+ *
+ * @param url Base URL
+ * @param params Object containing query parameters
+ * @param sorting Optional sorting parameters
+ * @returns URL with appended query parameters
+ */
+export const appendQueryParams = <T extends string>(
+  url: string,
+  params?: Record<string, any>,
+  sorting?: SortingOptions<T>,
+): string => {
+  const urlParams = new URLSearchParams();
+
+  if (params) {
+    Object.entries(params).forEach(([key, value]) => {
```

Comment:
Rewrite using `for ... of ...` instead of `.forEach`, since the last one is less performant comparing to the first.